README.md

```diff
@@ -26,13 +34,30 @@ For example:
 }
 ```
 
-This extension helps users add server definitions to their [user or workspace settings](https://code.visualstudio.com/docs/getstarted/settings) by editing JSON files.
+## Storing passwords securely
+
+This extension adds the command `InterSystems Server Manager: Store Password in Keychain` to the Command Palette, which offers a quickpick of defined servers, then prompts for a password to store. This facility should be used instead of the plaintext `password` property of a server's definition, which has been deprecated.
+
+**Usage:**
+
+1. Bring up the command palette by typing CTRL-Shift-P (Mac: Command-Shift-P)
```

Comment:
VS Code docs writes these as Ctrl+Shift+P / Cmd+Shift+P. It also capitalizes Command Palette. I suggest we do the same. There's a second occurrence later in your changes. Also, on Windows at least, F1 launches the Command Palette. Docs say the same happens on macOS. I find that a lot easier, and more memorable.